docs/assets/css/tabs.less

```diff
@@ -102,6 +97,13 @@
                     }
                 }
             }
+
+            &:first-child {
+                .govuk-tabs__tab {
```

Comment:
Hrm, customizing a third-party UI component smells bad.

Reply:
Yeaaaaaahhhh all of `tabs.less` is overrides for the [gov.uk tabs](https://design-system.service.gov.uk/components/tabs/). It smells real bad. I'd like to migrate them into our own atomic component if it's a pattern we intend on using elsewhere.